tests/publiccloud/patch_and_reboot.pm

```diff
@@ -22,8 +22,13 @@ sub run {
 
     my $remote = $args->{my_instance}->username . '@' . $args->{my_instance}->public_ip;
 
-    $args->{my_instance}->retry_ssh_command(cmd => "sudo zypper -n ref", timeout => 240, retry => 6);
+    my $cmd_time = time();
+    $args->{my_instance}->retry_ssh_command(cmd => "sudo zypper -n ref", timeout => (check_var('PUBLIC_CLOUD_PROVIDER', 'AZURE') ? 3600 : 240), retry => 6, delay => 60);
```

Comment:
please move out timeout calculation . it is too much for one line . hard to read and understand what is going on . e.g. : 
```
my $timeout = check_var('PUBLIC_CLOUD_PROVIDER', 'AZURE') ? 3600 : 240;
$args->{my_instance}->retry_ssh_command(cmd => "sudo zypper -n ref", timeout => $timeout, retry => 6, delay => 60);
```